### DISCUSSION TITLE
LumiFur App Support!

### DISCUSSION THREAD

**Discussion by stef1949** [coelacant1/ProtoTracer#33] [Ideas]
Hello! I am creating a native Apple SwiftUI app for iOS, MacOS, WatchOS & iPadOS!!! It's called LumiFur. I would love to get some help on the best way to communicate with your program!  ARFF ARFF!!! :3

https://github.com/stef1949/LumiFur

![nFKjiS](https://github.com/user-attachments/assets/6897b198-2b4b-4b56-9c77-ab0a470462e6)

**Comment by MeteorTheProto** [comment]
How does your app communicate with microcontrollers?

**Reply by stef1949** [reply]
I haven't implemented this yet but I'm planning on using Bluetooth LE

**Reply by stef1949** [reply]
> How does your app communicate with microcontrollers?

I have now implemented this 😌🐾🏳️‍⚧️
